2022-03-31 05:18:26,252 INFO    Thread-34 :29951 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-03-31 05:18:26,252 INFO    Thread-34 :29951 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-03-31 05:18:26,252 INFO    Thread-34 :29951 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'cloud classification', 'entity': 'leonardtd', 'root_dir': '/home/ltorres/leo/tesis/cloud-classification/code', 'run_id': 'w2nqk3lb', 'sweep_param_path': '/home/ltorres/leo/tesis/cloud-classification/code/wandb/sweep-03otgj81/config-w2nqk3lb.yaml', 'sweep_id': '03otgj81'}
2022-03-31 05:18:26,252 INFO    Thread-34 :29951 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-31 05:18:26,252 INFO    Thread-34 :29951 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_051826-w2nqk3lb/logs/debug.log
2022-03-31 05:18:26,252 INFO    Thread-34 :29951 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_051826-w2nqk3lb/logs/debug-internal.log
2022-03-31 05:18:26,252 INFO    Thread-34 :29951 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f684473ab20>
2022-03-31 05:18:26,253 INFO    Thread-34 :29951 [wandb_init.py:init():439] calling init triggers
2022-03-31 05:18:26,253 INFO    Thread-34 :29951 [wandb_init.py:init():442] wandb.init called with sweep_config: {'batch_size': 25, 'criterion': 'loge', 'device': 'cuda:2', 'epochs': 25, 'hidden_dim': 512, 'learning_rate': 3.0236125085020263e-05, 'num_heads': 1, 'num_hidden': 3, 'optimizer': 'adam', 'similarity_threshold': 0.9}
config: {}
2022-03-31 05:18:26,253 INFO    Thread-34 :29951 [wandb_init.py:init():492] starting backend
2022-03-31 05:18:26,253 INFO    Thread-34 :29951 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-31 05:18:26,260 INFO    Thread-34 :29951 [backend.py:ensure_launched():219] starting backend process...
2022-03-31 05:18:26,314 INFO    Thread-34 :29951 [backend.py:ensure_launched():224] started backend process with pid: 10836
2022-03-31 05:18:26,315 INFO    Thread-34 :29951 [wandb_init.py:init():501] backend started and connected
2022-03-31 05:18:26,334 INFO    Thread-34 :29951 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 25, 'criterion': 'loge', 'device': 'cuda:2', 'epochs': 25, 'hidden_dim': 512, 'learning_rate': 3.0236125085020263e-05, 'num_heads': 1, 'num_hidden': 3, 'optimizer': 'adam', 'similarity_threshold': 0.9}
2022-03-31 05:18:26,339 INFO    Thread-34 :29951 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-31 05:18:26,339 INFO    Thread-34 :29951 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-31 05:18:26,339 INFO    Thread-34 :29951 [wandb_init.py:init():565] updated telemetry
2022-03-31 05:18:26,437 INFO    Thread-34 :29951 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-31 05:18:27,206 INFO    Thread-34 :29951 [wandb_run.py:_on_init():1759] communicating current version
2022-03-31 05:18:27,278 INFO    Thread-34 :29951 [wandb_run.py:_on_init():1763] got version response 
2022-03-31 05:18:27,278 INFO    Thread-34 :29951 [wandb_init.py:init():625] starting run threads in backend
2022-03-31 05:18:28,751 INFO    Thread-34 :29951 [wandb_run.py:_console_start():1733] atexit reg
2022-03-31 05:18:28,751 INFO    Thread-34 :29951 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-31 05:18:28,752 INFO    Thread-34 :29951 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-31 05:18:28,753 INFO    Thread-34 :29951 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-31 05:18:28,753 INFO    Thread-34 :29951 [wandb_init.py:init():664] run started, returning control to user process
2022-03-31 07:31:39,808 INFO    Thread-34 :29951 [wandb_run.py:finish():1539] finishing run leonardtd/cloud classification/w2nqk3lb
2022-03-31 07:31:39,809 INFO    Thread-34 :29951 [jupyter.py:save_history():429] not saving jupyter history
2022-03-31 07:31:39,809 INFO    Thread-34 :29951 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 07:31:39,809 INFO    Thread-34 :29951 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-31 07:31:39,809 INFO    Thread-34 :29951 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-03-31 07:31:39,811 INFO    Thread-34 :29951 [wandb_run.py:_restore():1674] restore
2022-03-31 07:31:40,667 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 910
}

2022-03-31 07:31:40,968 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 910
}

2022-03-31 07:31:41,485 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 95220
}

2022-03-31 07:31:41,590 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 101663
}

2022-03-31 07:31:41,694 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 101663
}

2022-03-31 07:31:41,796 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 101663
}

2022-03-31 07:31:41,899 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 101663
}

2022-03-31 07:31:42,001 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 86258
  total_bytes: 101663
}

2022-03-31 07:31:42,105 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 95220
  total_bytes: 101663
}

2022-03-31 07:31:42,210 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 95220
  total_bytes: 101663
}

2022-03-31 07:31:42,326 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 95220
  total_bytes: 101663
}

2022-03-31 07:31:42,429 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}

2022-03-31 07:31:42,533 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}

2022-03-31 07:31:42,635 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}

2022-03-31 07:31:42,738 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}

2022-03-31 07:31:42,840 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}

2022-03-31 07:31:42,943 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}

2022-03-31 07:31:43,601 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}

2022-03-31 07:31:44,224 INFO    Thread-34 :29951 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 101663
  total_bytes: 101663
}
local_info {
}

2022-03-31 07:31:45,363 INFO    Thread-34 :29951 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-31 07:31:45,364 INFO    Thread-34 :29951 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-31 07:31:45,369 INFO    Thread-34 :29951 [wandb_run.py:_footer_sync_info():2822] logging synced files
